src/components/settings/SettingsHeaderRow.tsx

```diff
@@ -13,33 +13,27 @@ interface Props {
 
 /**
  * A blue header row in a settings scene.
+ * TODO: SettingsScene needs UI4 design.
  */
 export const SettingsHeaderRowComponent = (props: Props) => {
   const { icon, label } = props
   const theme = useTheme()
   const styles = getStyles(theme)
-  const colors = theme.settingsRowHeaderBackground
-  const start = theme.settingsRowHeaderBackgroundStart
-  const end = theme.settingsRowHeaderBackgroundEnd
 
   return (
-    <LinearGradient colors={colors} start={start} end={end} style={styles.row}>
+    <View style={styles.row}>
       {icon != null ? <View style={styles.padding}>{icon}</View> : undefined}
-      <Text style={styles.text}>{label}</Text>
-    </LinearGradient>
+      <EdgeText style={styles.text}>{label}</EdgeText>
+    </View>
   )
 }
 
 const getStyles = cacheStyles((theme: Theme) => ({
   row: {
-    // Layout:
-    minHeight: theme.rem(2.75),
-    padding: theme.rem(1),
-
-    // Children:
     alignItems: 'center',
     flexDirection: 'row',
-    justifyContent: 'flex-start'
+    justifyContent: 'flex-end',
```

Comment:
why `flex-end`? `flex-start` seems more appropriate

Reply:
You're right, but in this case it totally didn't matter either way because the
text fills the space while being left `textAligned`. Removed